cover_page/document.adoc

```diff
@@ -0,0 +1,61 @@
+= Title of IEC standard
+:comment: ### Circulation Cover Page template
+:comment: ### Indication of Closing date for voting
+:vote-ended-date: 2020-01-01
+:doctype: cover-page
```

Comment:
There is no such doctype; this should be the doctype of international-standard, as the cover page metadata is just affixed to such a document.

Reply:
Thanks @opoudjis, I have corrected it.
Do you think is there anything else to correct or enhance?